README.tr.md

```diff
@@ -0,0 +1,85 @@
+# sway
+
+[English][en] - [日本語][ja] - [Français][fr] - [Українська][uk] - [Español][es] - [Polski][pl] - [中文-简体][zh-CN] - [Deutsch][de] - [Nederlands][nl] - [Русский][ru] - [中文-繁體][zh-TW] - [Português][pt] - [Danish][dk] - [한국어][ko] - [Română][ro] - **[Türkçe][tr]**
+
+Sway, [i3]-uyumlu bir [Wayland] dizgicisidir. [SSS][FAQ]'yi okuyun. 
+[IRC kanalı][IRC channel]na katılın \(irc.freenode.net'te #sway (İngilizce)).
+
+## Sürüm imzaları
+
+Sürümler [E88F5E48] ile imzalandı ve [GitHub][GitHub releases]'da yayınlandı.
+
+## Kurulum
+
+### Paketler ile
+
+Sway birçok dağıtımda mevcuttur. Sizinki için "sway" paketini yüklemeyi deneyin.
+
+Dağıtımınız için sway'i paketlemekle ilgileniyorsanız, IRC kanalına uğrayın veya tavsiye için sir@cmpwn.com adresine bir e-posta gönderin.
+
+### Kaynak koddan derleme
+
+Test veya geliştirme için sway ve wlroots'un HEAD'ini oluşturmak istiyorsanız [bu wiki sayfası][Development setup]na göz atın.
+
+Aşağıdaki bağımlılıkları yükleyin:
+
+* meson \*
+* [wlroots]
+* wayland
+* wayland-protocols \*
+* pcre
+* json-c
+* pango
+* cairo
+* gdk-pixbuf2 (isteğe bağlı: system tray)
+* [scdoc] (isteğe bağlı: man pages) \*
+* git (isteğe bağlı: version info) \*
+
+_\*Derleme-anı bağımlılıkları_
+
+Şu komutları çalıştırın:
+
+    meson build
+    ninja -C build
+    sudo ninja -C build install
+
+logind olmayan sistemlerde, sway ikilisine(binary) izin vermeniz (suid) gerekir:
```

Comment:
Same here: is there a missing space before the parenthesis?

Reply:
ah yes, should be